sync/class.jetpack-sync-module-comments.php

```diff
@@ -40,6 +43,23 @@ public function init_listeners( $callable ) {
 		$this->init_meta_whitelist_handler( 'comment', array( $this, 'filter_meta' ) );
 	}
 
+	/**
+	 * @param array $data       The new, processed comment data.
+	 * @param array $comment    The old, unslashed comment data.
+	 */
+	public function handle_comment_contents_modification( $data, $comment ) {
```

Comment:
$data is a very generic variable name. Can we make it more explicit `$new_comment`.